src/msg/message.cc

```diff
@@ -5,12 +5,16 @@
 
 #include <boost/functional/hash.hpp>
 
+#include <mutex>
+#include <shared_mutex>
 #include <typeindex>
 #include <unordered_map>
 #include <utility>
 
 namespace cris::core {
 
+using channel_id_t = CRMessageBase::channel_id_t;
```

Comment:
Wonder if you can say `using CRMessageBase::channel_id_t;`

Reply:
No

```
src/msg/message.cc:16:22: error: using declaration cannot refer to class member
using CRMessageBase::channel_id_t;
      ~~~~~~~~~~~~~~~^
```